package/Dockerfile

```diff
@@ -57,7 +57,7 @@ ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 102.0.0+up0.4.0
 # System charts minimal version
 ENV CATTLE_FLEET_MIN_VERSION=102.1.0+up0.7.0-rc.2
 ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=2.0.5+up0.3.5-rc4
-ENV CATTLE_CSP_ADAPTER_MIN_VERSION=2.0.1
+ENV CATTLE_CSP_ADAPTER_MIN_VERSION=2.0.2+up2.0.2-rc2
```

Comment:
I still don't understand with charts, in general, why, for example, this can't be `2.0.2-rc2`.

Reply:
Before +up defines the version of rancher that it's compatible with (i.e. each release this will go up by one). After +up defines the actual app version. At least, that's how I remember it.

Adapter gets to somewhat bypass this because we sync these values ok.

Reply:
But the part before +up, `2.0.2`, is not a Rancher version...

Reply:
Looks like I'm wrong - before the +up defines the chart version as it's known to rancher, after defines how it's known to upstream - https://github.com/rancher/charts/blob/dev-v2.7/docs/developing.md#version. For the adapter this is the same except when it is in the RC state.

Reply:
Thanks for the clarification!